netbout-client/pom.xml

```diff
@@ -180,7 +180,12 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
-                                <exclude>findbugs:.*</exclude>
+                                <!--
+                                @todo #911 Fix findbugs errors in client
```

Comment:
@dmzaytsev whole puzzle description is bigger than a single line. I have no idea what to write here. Is it somehow unclear? Formatting is clear, will fix it